dev_guide/integrating_external_services.adoc

```diff
@@ -11,17 +11,18 @@ toc::[]
 
 == Overview
 
-Many OpenShift applications use external resources, such as external databases, or an external
-SaaS endpoint. These external resources can be modeled as native OpenShift services, so that
-applications can work with them as they would any other internal service.
-
+Many OpenShift applications use external resources, such as external databases,
+or an external SaaS endpoint. These external resources can be modeled as native
+OpenShift services, so that applications can work with them as they would any
+other internal service.
 
 == External MySQL Database
 
-One of the most common types of external services is an external database. To support an external database, an application needs:
+One of the most common types of external services is an external database. To
```

Comment:
@tnguyen-rh I see what you mean, but I feel it's best to keep it in there, if only to differentiate between this and internal services. I know they're not available in OpenShift, but hey, they might one day.

Also, the title of the topic is "Integrating with External Services", so it might be wise to keep with the point.

Willing to haggle if you disagree :frog: